SetLibrary/SetLibrary/SetLibrary/SetWriter.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using StreamWriter = System.IO.StreamWriter;
+
+
+
+namespace SetLibrary.Serialize
+{
+	public class SetWriter : IDisposable
+	{
+        private bool disposedValue;
+        StreamWriter streamWriter;        
+
+        public SetWriter(string filePath)
+		{
+            streamWriter = new(filePath);
+		}
+
+        public void Serialize(NumSet set)
+        {
+
+        }
+
+        protected virtual void Dispose(bool disposing)
+        {
+            if (!disposedValue)
+            {
+                if (disposing)
+                {
+                    // TODO: dispose managed state (managed objects)
+                    streamWriter.Close();
```

Comment:
```suggestion
                    streamWriter.Dispose();
```
You should use `Dispose` here to permanently close the memory connection. `Close` may eventually call `Dispose` but it has the potential to re-open itself as all resources are not cleaned up right away.